src/transforms/push.ts

```diff
@@ -106,110 +106,103 @@ export function processPushJob(app: Application) {
 }
 
 export const processPush = async (github: GitHubAPI, payload, rootLogger: LoggerWithTarget) => {
```

Comment:
The only thing I did here is removing try/catch block
```
} catch (err) {
		log.error({ err }, "Failed to process push");
		emitWebhookFailedMetrics("push");
		throw err;
}
```

We don't need to do it here because:
1. If this function fail in the middleware during webhook processing, we already emit metrics in the middleware error handling (currently we emitting failure metrics twice)
2. For SQS we'll have its own error handling. (Otherwise we'll emit a failure on each retry)
3. We don't use Redis for pushes anymore, so we won't loosing its metrics